tests/test_data.py

```diff
@@ -623,3 +623,14 @@ def test_multiple_vars(self):
         [draw_mA, draw_mB] = pm.draw([mA, mB])
         assert draw_mA.shape == (10,)
         np.testing.assert_allclose(draw_mA, -draw_mB)
+
+
+def test_scaling_data_works_in_likelihood() -> None:
+    data = np.array([10, 11, 12, 13, 14, 15])
+
+    with pm.Model() as model:
+        target = pm.Data("target", data)
+        scale = 12
+        scaled_target = target / scale
+        mu = pm.Normal("mu", mu=0, sigma=1)
+        pm.Normal("x", mu=mu, sigma=1, observed=scaled_target)
```

Comment:
Should I sample this to check that it has the correct data in the InferenceData?

Reply:
Well maybe. Just make sure to do a cheap sampling, since we don't care about draws at all?

Reply:
I've tested the "extract_..." function directly

Reply:
Yeah that's a more direct unit test, this would be a CI, we don't care how it's done just want to be sure the data is there in the end?

Reply:
I still want some sampling here to check the outputs? This test is not testing anything explicitly at the moment

Reply:
I will add sampling to this test when I can

Reply:
done